airflow/providers/google/cloud/hooks/cloud_sql.py

```
@@ -407,6 +408,51 @@ def _wait_for_operation_to_complete(self, project_id: str, operation_name: str)
 )
 
 
+class CloudSQLAsyncHook(GoogleBaseAsyncHook):
+    """Class to get asynchronous hook for Google Cloud SQL."""
+
+    sync_hook_class = CloudSQLHook
+
+    async def get_conn(self, session, url):
+
+        scopes = [
+            "https://www.googleapis.com/auth/cloud-platform",
+            "https://www.googleapis.com/auth/sqlservice.admin",
+        ]
+
+        async with Token(scopes=scopes) as token:
+            session_aio = AioSession(session)
+
+            headers = {
+                "Authorization": f"Bearer {await token.get()}",
+            }
+
+            try:
+                operation = await session_aio.get(url=url, headers=headers)
+            except AirflowException:
+                pass
```

why we are ignoring error?